content/v2.0/reference/cli/influx/_index.md

```diff
@@ -62,9 +62,15 @@ retrieving authentication tokens._
 | [user](/v2.0/reference/cli/influx/user)           | User management commands                             |
 | [write](/v2.0/reference/cli/influx/write)         | Write points to InfluxDB                             |
 
-{{% influx-cli-global-flags %}}
+## Mapped environment variables
+Some `influx` CLI flags are mapped to environment variables.
```

Comment:
Do we have list of these mapped flags (or how to search for/find)?

Reply:
That's what this PR addresses. All flags that can be mapped now have a column in the flags table that shows the environment variable. There's a link the column label that brings them back here.